apps/video/js/video.js

```diff
@@ -1386,8 +1386,9 @@ function showThrobber() {
 }
 
 function hideThrobber() {
-  dom.throbber.classList.add('hidden');
   dom.throbber.classList.remove('throb');
+  // setTimeout is a workaround for Bug 1166500
+  setTimeout(function() { dom.throbber.classList.add('hidden'); }, 100);
```

Comment:
I notice you didn't do anything for bug 962594 for the video app. I'm not super-familiar with the video app, so I'm not sure you actually need to do anything, but I just wanted to be sure.